docs/templates/nextflow_dsl1/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nfcore/base:1.13.3
+LABEL authors="your name"
+      description="Docker image containing all software requirements for the [execution_tool] pipeline"
+
+# Install the conda environment
+COPY environment.yml /
+RUN conda env create --quiet -f /environment.yml && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/[execution_workflow]/bin:$PATH
```

Comment:
This can potentially lead to problems. First, this particular image already has `conda activate base` in the `/root/.bashrc`, and that will prepend to `PATH` during shell sessions (i.e., take precedence over the environment). Second, not using `conda activate` fails to run activation scripts, which can lead to undefined behavior.

Better practice here would be to remove `conda activate base` and replace it with `conda activate [execution_workflow]`. I'd combine this with above to:

```bash
RUN conda env create -qf /environment.yml && \
    sed -i 's/^conda activate base$/conda activate [execution_workflow]/' ~/.bashrc && \
    conda clean -qafy
```

Reply:
Thanks! I have updated the line accordingly!